.github/workflows/pull-request-build.yaml

```diff
@@ -0,0 +1,54 @@
+name: Build SDK
```

Comment:
non-blocker: Switch to "CLI" or give a more descriptive name, line 20 too?

Something with "internal", "installers", "integration tests", etc.